### PR TITLE
[IMP] mail: introduce AttachmentViewer/onClickClose

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2579306

### CODE REVIEW

File: addons/mail/static/src/models/attachment_viewer.js [hunk 1]
```diff
@@ -16,6 +16,14 @@ registerModel({
         containsElement(element) {
             return Boolean(this.component && this.component.root.el && this.component.root.el.contains(element));
         },
+        /**
+         * Called when clicking on cross icon.
+         *
+         * @param {MouseEvent} ev
+         */
+        onClickClose(ev) {
+            this.delete();
```
Comment (phenix-factory) [inline addons/mail/static/src/models/attachment_viewer.js:25]: This will need to use the model close method.

**Review by phenix-factory — APPROVED** [review]
Small rebase stuff, otherwise LGTM

**Comment by alexkuhn** [comment]
Will r-plus when previous commit is deployed

**Comment by seb-odoo** [comment]
obsolete pr

**Pull request closed without merging**